edu.stanford.protege.appdata.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+<component>
```

Comment:
This needs to include `type="desktop-application"`, otherwise it won't be visible on the website.

Reply:
Good catch! I'll fix it in the new repo.